portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,92 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let root = document.documentElement;
```

Comment:
const, this never changes.  But also you only use it in one place so it doesn't need to be a separate variable, you can just inline it.